warpgate-db-entities/src/TargetGroup.rs

```diff
@@ -0,0 +1,25 @@
+use poem_openapi::Object;
+use sea_orm::entity::prelude::*;
+use serde::Serialize;
+use uuid::Uuid;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, Serialize, Object)]
+#[sea_orm(table_name = "target_groups")]
+#[oai(rename = "TargetGroup")]
+pub struct Model {
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub id: Uuid,
+    pub name: String,
+    #[sea_orm(column_type = "Text")]
+    pub description: String,
+    pub color: Option<String>, // Hex color code for UI display
```

Comment:
For cleaner UI, please restrict the color to existing bootstrap theme-colors (`primary`, `secondary`, `danger` etc) - this lets you plug into existing components (e.g. `badge`) and avoids the need to manually generate contrasting font colors etc